genai-on-vertex-ai/vertex_evaluation_services/gemini-curate-evaluation-data/curate_new_evals.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        task: str = "RETRIEVAL_DOCUMENT",
</pre><p>Can we add a line explaining what is TASK_TYPE and why RETRIEVAL_DOCUMENT was chosen here ? let's also link it back to our docs - <a href="https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/task-types#retrieve_information_from_texts" target="_blank">https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/task-types#retrieve_information_from_texts</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>